content/docs/rkvst-basics/grouping-assets-by-location/index.md

```diff
@@ -206,13 +204,68 @@ curl -v -X POST \
 {{< /tab >}}}
 {{< /tabs >}}
 
+6. View your Locations. 
+
+ {{< tabs name="location_list_locations" >}}
+ {{{< tab name="UI" >}}
+ Navigate to `Manage Locations` in the Sidebar to see a list of existing Locations.
+ {{< img src="LocationView.png" alt="Rectangle" caption="<em>Managing a Location</em>" class="border-0" >}}
+ {{< /tab >}}
+ {{< tab name="YAML" >}}
+ You can view all Location data using the `LOCATIONS_LIST` action. Use the `print_response` keyword to get the full output. 
+
+ ```yaml
+ ---
+ steps:
+   - step:
+       action: LOCATIONS_LIST
+       description: List all locations.
+       print_response: true
+ ```
+{{< /tab >}}
+{{< tab name="JSON" >}}
+Use the [Locations API](https://docs.rkvst.com/docs/api-reference/locations-api/) to GET a list of existing locations.
+ 
+```bash
+curl -v -X GET \
+     -H "@$BEARER_TOKEN_FILE" \
+     https://app.rkvst.io/archivist/v2/locations
+```
+{{< /tab >}}}
+{{< /tabs >}}
 
-6. Navigate to `Manage Locations` in the Sidebar to see a list of existing Locations.
-{{< img src="LocationView.png" alt="Rectangle" caption="<em>Managing a Location</em>" class="border-0" >}}
-
-7. You can inspect details of a single Location. Click the desired Location row. 
-{{< img src="LocationDetails.png" alt="Rectangle" caption="<em>Viewing a Location</em>" class="border-0" >}}
+7. View details of the Location you just created. 
+
+ {{< tabs name="specific_location_locations" >}}
+ {{{< tab name="UI" >}}
+ You can inspect details of a single Location. Click the desired Location row. 
+ {{< img src="LocationDetails.png" alt="Rectangle" caption="<em>Viewing a Location</em>" class="border-0" >}}
+ {{< /tab >}}
+ {{< tab name="YAML" >}}
+ The `LOCATIONS_LIST` action can be filtered using identifying attributes (`attrs`) to view the details of a specific Location.
```

Comment:
This component also has a URL - https://python.rkvst.com/runner/components/locations_list.html which is the general description.
This could be used (what do you think?) but if so we would have to coordinate when changes are made to the python docs...

Reply:
I'd like to avoid linking to the python docs in this section, as it is written for a target audience of someone unfamiliar/new to with RKVST. The links I use throughout all stay within docs.rkvst.com, so the reader doesn't have to leave that one source. I think the python docs may contain more information than someone starting out with the yaml runner needs.

Reply:
Agreed. I dislike duplication but I dislike tight coupling more. The YAML Runner shouldn't be tethered to the pysdk from a user perspective.